Library/Homebrew/tap.rb

```diff
@@ -82,6 +82,16 @@ def git?
     (path/".git").exist?
   end
 
+  # The issues URL of this {Tap}.
+  # e.g. `https://github.com/user/homebrew-repo/issues`
+  def issues_url
+    if official?
+      "https://github.com/Homebrew/homebrew-#{repo}/issues"
+    else
+      "https://github.com/#{name}/issues"
```

Comment:
You don't need to check `official?`, just use `"https://github.com/#{user}/homebrew-#{repo}/issues"` directly.

Also FYI, `name` variable doesn't include `homebrew-` as well.